packages/tasks/src/pipelines.ts

```diff
@@ -635,6 +635,16 @@ export const PIPELINE_DATA = {
 		modality: "cv",
 		color: "yellow",
 	},
+	"text-to-3d": {
```

Comment:
We need to tag https://huggingface.co/bennyguo/zero123-xl-diffusers with this type, feel free to add a PR! We should similarly start adding PRs to add models to https://huggingface.co/models?other=image-to-3d